src/content/docs/policy/repository_policy.md

```diff
@@ -6,47 +6,79 @@ CachyOS Repository Usage Policy
 
-## 2. Authorized Users
+## 2. Disclaimer of Warranty
 
-The CachyOS repository is exclusively available for the following users:
+Unless required by applicable law or agreed to in writing, the CachyOS team provides the repository on an **"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND**, either express or implied, including, without limitation, 
+any warranties or conditions of **TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE**. You, as repository users, are solely responsible for determining the appropriateness of using the repository and assume
+any risks associated with your exercise of permissions.
 
-CachyOS Users
-ArchLinux Users
+## 3. Limitation of Liability
 
-## 3. Unauthorized Use
+In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise, unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing, shall the CachyOS team 
+be liable to repository users for damages, including any direct, indirect, special, incidental, or consequential damages of any character arising as a result of this policy or out of the use or inability to use the repository 
+(including but not limited to damages for loss of goodwill, work stoppage, computer failure or malfunction, or any and all other commercial damages or losses), even if the CachyOS team has been advised of the possibility of such damages.
 
-Use of the CachyOS repository by any other Linux distributions, including other Arch-based distributions, is strictly prohibited. This includes, but is not limited to:
+## 4. User Scope
 
-Manjaro
-EndeavourOS
-ArcoLinux
-Parabola
-Any other Linux distribution not explicitly mentioned in the "Authorized Users" section
+The CachyOS repository (and its mirrors) is designed exclusively for the following users:
 
-# 4.Mirror
+- CachyOS Users
+- ArchLinux Users
+
+## 5. Unsupported Users
+
+Users from other distributions are **NOT SUPPORTED** and **HIGHLY DISCOURAGED** to use the CachyOS repository (and its mirrors). This includes, but is not limited to:
+
+- Manjaro
+- EndeavourOS
+- ArcoLinux
+- Parabola
+- Users of any other Linux distribution not explicitly mentioned in the "User Scope" section.
+
+## 5. Redistribution of the Repository
+
+By "redistribution", we mean the behaviors of inclusion of the CachyOS repository (and its mirrors) or packages obtained from the CachyOS repository as a part of the distributed image of the operating system. 
+This also applies to the behaviors of providing the utilities that enable CachyOS repository by users' choice, or providing any distributed or official document that guide users to enable CachyOS 
+repository (and its mirrors) by their means. 
+
+The CachyOS repository is exclusively authorized for redistribution in the following distributions:
+
+- CachyOS
+- ArchLinux
```

Comment:
I don't think we should put ArchLinux here

Reply:
fixed